html5/js/Client.js

```diff
@@ -942,7 +942,7 @@ class XpraClient {
       //this usually fires when we have received the event via "oninput" already
       return;
     }
-    let str = event.key || String.fromCharCode(keycode);
+    let string_ = event.key || String.fromCharCode(keycode);
```

Comment:
`str` is not a reserved word, but it's generally an unnecessary abbreviation and specifically (here) a vague variable name. I don't know why the linter added the `_` suffix, but I've renamed the variable to `keystring` (hopefully a contextually accurate name).

Reply:
I'm going to close this, assuming that you like the "underscore suffixes are fine but renaming to something more descriptive is better" solution. I really like it, since I think abbreviations are usually unnecessary, vague, and confusing.